.github/workflows/linux.yml

```diff
@@ -56,32 +55,35 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.9]
+        python-version: [3.10]
 
     steps:
     - uses: actions/checkout@v2
 
-    - name: Install Python 3.9
-      if: matrix.python-version == '3.9'
+    - name: Install Python 3.10
+      if: matrix.python-version == '3.10'
       run: |
         sudo add-apt-repository ppa:deadsnakes/ppa
         sudo apt-get update
-        sudo apt-get install -y --no-install-recommends python3.9-dev python3.9-distutils python3.9-venv
-        python3.9 -m pip install --upgrade pip setuptools
+        sudo apt install -y --no-install-recommends python3.10-dev python3.10-distutils python3.10-venv
+        curl -O https://bootstrap.pypa.io/get-pip.py && python3.10 get-pip.py && rm get-pip.py
```

Comment:
So if this is necessary, it's a (temporary I guess) issue with the Python 3.10 that GitHub currently provides. Maybe add a comment that says that and points to this PR. We should be able to remove this later on.